autogen

```diff
@@ -221,6 +226,11 @@ while :; do
       TOP_LEVEL_COMMENT=0
       ;;
 
+    --no-pycode | --no-pyc)
```

Comment:
We should also add some tests for new functionality to ensure it does what we think / expect it to do.